bom/runtime/pom.xml

```diff
@@ -72,17 +80,15 @@
         <commons-logging-jboss-logging.version>1.0.0.Final</commons-logging-jboss-logging.version>
         <commons-lang3.version>3.9</commons-lang3.version>
         <commons-codec.version>1.13</commons-codec.version>
-        <validation-api.version>2.0.1.Final</validation-api.version>
         <classmate.version>1.3.4</classmate.version>
-        <javax.el-impl.version>3.0.1-b11</javax.el-impl.version>
-        <hibernate-validator.version>6.1.0.Alpha6</hibernate-validator.version>
+        <jakarta.el-impl.version>3.0.3</jakarta.el-impl.version>
```

Comment:
Should this be moved to where all the other `jakarta.*` properties are?

It can be done in a later PR if it's easier

Reply:
Yeah, it used to be there because it was added with HV. But I can see how it makes sense to move them all at the same place now.

Reply:
I reordered things.